### PR TITLE
Create anon set DB

### PR DESCRIPTION
**Pull request by lsankar4033**
Database that serves as our store of all anonymity sets.

Currently deployed to render with a read-replica for frontends like creddd.xyz

### CODE REVIEW

File: packages/anon-set-db/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+model AnonSetMerkleTree {
+  merkleRoot String @id
+
+  metadata String // NOTE: i.e. versioning info, block num, etc.
+
+  addressesWithMerkleProof AddressWithMerkleProof[]
+  AnonSet                  AnonSet?                 @relation(fields: [anonSetId], references: [id])
+  anonSetId                String?
```
Comment (DanTehrani) [inline packages/anon-set-db/prisma/schema.prisma:30]: @lsankar4033  Do you think we should make this a required foreign key? (remove the "?")

File: packages/anon-set-db/prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  address     String
+  path        String[]
+  pathIndices Int[]
+
+  AnonSetMerkleTree           AnonSetMerkleTree? @relation(fields: [anonSetMerkleTreeMerkleRoot], references: [merkleRoot])
+  anonSetMerkleTreeMerkleRoot String?
```
Comment (DanTehrani) [inline packages/anon-set-db/prisma/schema.prisma:43]: @lsankar4033 here too

**Comment by DanTehrani** [comment]
@lsankar4033 
The schema looks good!
I think we can add these tables to the credmapper-db (future "cred" db) as part of the consolidation: https://github.com/personaelabs/credmapper/pull/4

File: packages/anon-set-db/prisma/schema.prisma (hunk 2 above)
Comment (lsankar4033) [reply packages/anon-set-db/prisma/schema.prisma:43]: yes on both

**Comment by lsankar4033** [comment]
> @lsankar4033 The schema looks good! I think we can add these tables to the credmapper-db (future "cred" db) as part of the consolidation: [personaelabs/credmapper#4](https://github.com/personaelabs/credmapper/pull/4)

still tend to think it's worth splitting this out as the read/write logic is different (multi

**Pull request merged**